.travis.yml

```diff
@@ -10,12 +10,14 @@ env:
 before_install:
   - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
   - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo add-apt-repository ppa:duggan/bats --yes
   - sudo apt-get update
-  - sudo apt-get -y install docker-ce
+  - sudo apt-get -y install docker-ce bats
 
 script:
   - docker build -t $DOCKER_IMAGE_NAME ./build
   - ./test/run.sh
+  - bats tests
```

Comment:
I'd rather not add some new tests using `bats` in this PR but add them after the `0.8` release. I know this is not a problem in general, but it keeps the codebase clean and consistent after all. Could you port the `bats` tests into our current test-suite, without using `bats` in this PR?

Reply:
I can remove all the tests from bats and add them later in other PR.

Reply:
Thanks! :)